lib/instrumentation/@azure/functions.js

```diff
@@ -47,6 +48,36 @@ module.exports = function initialize(agent, azureFunctions, _moduleName, shim, {
     return
   }
 
+  // Wrap app.setup to check if streaming is enabled.
+  // `setup` is an Object property of `azureFunctions.app`
+  // and calls Azure's underlying src/setup.ts `setup` function.
+  const originalSetupDescriptor = Object.getOwnPropertyDescriptor(azureFunctions.app, 'setup')
```

Comment:
you can `shim.wrap` this method and check if that way. we should not be diverging from our standard ways of instrumenting